Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -294,8 +294,11 @@ static u32 AdvanceDTK(u32 maximum_samples, u32* samples_to_process)
 static void DTKStreamingCallback(DIInterruptType interrupt_type, const std::vector<u8>& audio_data,
                                  s64 cycles_late)
 {
+  // TODO: Should we use GetAISSampleRate instead of Get48KHzSampleRate?
```

Comment:
I'm assuming you intend to investigate this at a later date?

Reply:
It would involve some kind of hardware testing that I'm not sure how to do. The issue (if it even is an issue) is present in master as well, so I added the comment just for clarity.

Reply:
I think the comment should expand on why we might want to use `GetAISSampleRate` instead, so anyone else that wants to investigate in some form has all the reasoning behind the TODO.

Reply:
I've reworded the comment in a way that hopefully should make the reason clearer.